dfsclient/src/main/java/org/hpccsystems/dfs/client/HPCCFile.java

```diff
@@ -250,6 +250,10 @@ private void updateProjectedRecordDef() throws Exception
                 field.setSourceType(HpccSrcType.LITTLE_ENDIAN);
             }
 
+            // if (field.isBlob())
+            // {
+            //     field.setIsBlob(false);
+            // }
         }
```

Comment:
is this commented out code needed?

Reply:
Yep, I have commented it back in as part of the subsequent commit